### PR TITLE
Update setup.py + black formatting

### PR DESCRIPTION
**Pull request by A-CGray**
## Purpose

pyHyp doesn't currently work with any numpy past 1.23 (#69), this PR reflects this in the installation requirements

## Expected time until merged

1 hour

## Type of change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Checklist

- [ ] I have run `flake8` and `black` to make sure the Python code adheres to PEP-8 and is consistently formatted
- [ ] I have formatted the Fortran code with `fprettify` or C/C++ code with `clang-format` as applicable
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

**Comment by A-CGray** [comment]
@sseraj, Marco mentioned that pyhyp actually works with 1.23? If that's the case I'll leave it up to you whether you want to bump the required version

**Review by marcomangano — DISMISSED** [review]

**Comment by sseraj** [comment]
Yes, pyHyp works with 1.23. I will update the PR.

**Review by sseraj — APPROVED** [review]

**Review by marcomangano — APPROVED** [review]

**Comment by marcomangano** [comment]
@sseraj please go ahead and merge, I am not a maintainer.

**Pull request merged**
